runtime/src/main/java/org/corfudb/runtime/collections/TxnContext.java

```diff
@@ -451,6 +451,30 @@ K enqueue(@Nonnull Table<K, V, M> table,
         return ret;
     }
 
+    /**
+     * Apply a list of Corfu SMREntries directly to a stream. This can be used for replaying the mutations
```

Comment:
can you update the javadoc for this method?  We should also add the reason why this kind of a wrapper is required, i.e., avoiding materializing the queue in memory.

Also, this method can be renamed to something like logUpdateEnqueue() or something similar.

Reply:
Updated.

Reply:
Updated!